packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -44,52 +28,52 @@ export type QueryContextType<
   removeQueryFilter: ImmutableTableQueryResult['removeQueryFilter']
   /** Removes a value from a QueryFilter. If no more values remain in the filter, the filter is also removed */
   removeValueFromQueryFilter: ImmutableTableQueryResult['removeValueFromQueryFilter']
-  /** The error returned by the query request, if one is encountered */
-  error: SynapseClientError | null
```

Comment:
Removed `error` from QueryWrapper, since each component manages its own query requests. They can throw on error which will be caught by `QueryWrapperErrorBoundary`.